### ISSUE TITLE
Always crash when play the VR Sample scene

### ISSUE THREAD

**Issue by duanyunlun** [ValveSoftware/steamvr_unity_plugin#808]
1,Dev Environment: Unity 2019.4.8f1 LTS ，with these packages
![image](https://user-images.githubusercontent.com/15923235/90089496-1d7dda80-dd54-11ea-9d8e-dec04a44a3ea.png)

2,Hardware : i9 9900K + RTX 2080 + 64G RAM (2666Mhz)

3,Frequency of occurrence : everytime

4,Steps to reproduce: 
   1) Download and import OpenVR-XR-Plugin-Installer.unitypackge and steamvr_2_6_1.unitypackage from offical github sites;
   2) Open UnityHub to create a new project with URP template;
   3) Import 1)downloaded packages;
   4) Generate SteamVR Input modules;
   5) Open the SteamVR Interactable Sample Scene and play, then Crash. **Actually ANY SCENE WITH VR MODULE WILL CRASH!**

![image](https://user-images.githubusercontent.com/15923235/90089317-a9433700-dd53-11ea-855c-74f54150aa6b.png)

**Comment by duanyunlun** [comment]
**Here is a workaround:** 
1, Open any Scripts in the Project, then press "Ctrl + Alt + P" to Attach to the Unity.exe process;
![image](https://user-images.githubusercontent.com/15923235/90089909-3a66dd80-dd55-11ea-9d6a-f3e0b9369090.png)
![image](https://user-images.githubusercontent.com/15923235/90089928-45ba0900-dd55-11ea-8df2-d69b4aaf77b6.png)

2, Back to Unity and click Play, you shell get the Debugger reaction，like this:
![image](https://user-images.githubusercontent.com/15923235/90090004-7601a780-dd55-11ea-8dc3-4a8ca5f0136d.png)

3, Then stop the Debugger, Unity will keep running rather than CRASH WITHOUT WARNING.

PS: Actually there is an ErrorLog always shows after Click Play, like below:
![image](https://user-images.githubusercontent.com/15923235/90090091-b8c37f80-dd55-11ea-946a-c5a1e6a8a8f7.png)
![image](https://user-images.githubusercontent.com/15923235/90090113-c2e57e00-dd55-11ea-9ca4-805a9b73acbc.png)
![image](https://user-images.githubusercontent.com/15923235/90090193-00e2a200-dd56-11ea-86a2-76f15cbdbe64.png)

It's an ERROR that shows the Teleport.instance NULL when other script try to get it, I don't know if there is a relationship between the two。

**Comment by zite** [comment]
This seems like it's probably a unity issue. As you can see, instance
should never be null in that scene.

**Comment by duanyunlun** [comment]
> This seems like it's probably a unity issue. As you can see, instance should never be null in that scene.
In fact, The UpdateVisuals() was called in the TeleportPoint.cs's Awake() Method, When I move it to the Start() Method, these ERROR no longer occurs
![image](https://user-images.githubusercontent.com/15923235/90101270-fbdf1c00-dd70-11ea-9ee2-00eea8259a34.png)

**Comment by nhf75** [comment]
Try to set OpenVr Stereo Rendering Mode to Multi Pass and Mirror View Mode to None.

**Comment by duanyunlun** [comment]
> Try to set Rendering Path to Foward in your Camera, OpenVr Stereo Rendering Mode to Multi Pass and Mirror View Mode to None.

But this loses the meaning of using URP lol

**Comment by nhf75** [comment]
Yes, but this the only way to stop crash to continue you development until new release come. Unfortunately.

**Comment by zite** [comment]
When instance is called, if it is null for some reason it should scan the hierarchy for any teleport object and use that. If you have a teleport object in your scene then instance should never be null.
